Service/Model.SqlCE/TodoEntityMapper.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Drawing;
+using Interface = TodoSystem.Service.Model.Interface;
+
+namespace Model.SqlCe
+{
+    /// <summary>
+    /// Mapper class for conversion from entity framework to dto.
+    /// </summary>
+    public static class TodoEntityMapper
```

Comment:
Yes, this mapper was temporal decision firstly. Stackoverflow participants proposed automapper  as best solution but I postponed it usage until to detail understanding.